src/endpoint.cpp

```diff
@@ -541,13 +541,52 @@ void Endpoint::log_aggregate(unsigned int interval_sec)
     }
 }
 
+UartEndpoint::UartEndpoint(std::string name)
+    : Endpoint{ENDPOINT_TYPE_UART, std::move(name)}
+{
+    ; // nothing else to do here
```

Comment:
and no need for an empty statement neither

Reply:
OK, I'll leave the comment tough, so it's clear, that the c-tor is intentionally empty.